pygmt/datasets/earth_age.py

```diff
@@ -26,6 +26,13 @@ def load_earth_age(resolution="01d", region=None, registration=None):
     registration type (**p** for pixel registration or **g** for gridline
     registration).
 
+    The default CPT for this dataset is *@earth_age.cpt*. It's implicitly used
```

Comment:
Please help improve the description of the default CPT before I make similar changes for other datasets.